R/series__series.R

```diff
@@ -251,6 +251,15 @@ Series_rem = function(other) {
   .pr$Series$rem(self, wrap_s(other))
 }
 
+#' @param index index of the Series (0-base)
+#' @param str_length length of string to show
+#' @return a string
+#' @examples
+#' pl$Series(c("foo", "bar"))$get_fmt(index = 1, str_length = 3)
+#' @noRd
+Series_get_fmt = function(index, str_length) {
```

Comment:
if we define this function `get_fmt()` it will be public. If we want it to stay private we can just, skip this definition, and access from anywhere else internally with `.pr$Series$get_fmt`. Currently `.pr$...` is exported such that users could access the private methods, but we could remove this export such that a user must write `polars:::.pr$...` to use them.

Reply:
we can also just remove the  `Series_` prefix, then it will not be exported as a public method